README.md

```diff
@@ -74,7 +74,225 @@ The Performance Working Group uses [GitHub issues](https://github.com/expressjs/
 The discussions are open to the public and anyone may participate. Also, the group uses the channel `#express-perf-wg`
 in the [OpenJS Foundation Slack](https://openjsf.org/collaboration) for realtime discussions.
 
+## How to Run the Load Tests
+
+### Requirements
+- Node.js v22 or higher
```

Comment:
We should try to make it clear that the tooling should be able to run a test with older node versions. Maybe something like this?

```suggestion
- Node.js v22 or higher (tests can still specify older Node.js versions)
```